music21/romanText/writeRoman.py

```diff
@@ -0,0 +1,348 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+# Name:         romanText/writeRoman.py
+# Purpose:      Writes Roman text files given a music21 stream of Roman numerals
+#
+# Authors:      Mark Gotham
+#
+# Copyright:    Copyright © 2020 Michael Scott Cuthbert and the music21 Project
+# License:      BSD, see license.txt
+# ------------------------------------------------------------------------------
+'''
+Writes Roman text files given a music21 stream of Roman numerals
```

Comment:
RomanText -- should have a reference to the specification format (which I believe we know the authors...)